diff --git a/packages/script/src/index.ts b/packages/script/src/index.ts
index 50edcb9..9b60a53 100644
--- a/packages/script/src/index.ts
+++ b/packages/script/src/index.ts
@@ -11,8 +11,8 @@ if (!expectedBunVersion) {
 }
 
 // relax version requirement
-const expectedBunVersionRange = `^${expectedBunVersion}`
+const expectedBunVersionRange = ">=1.3.13 <2.0.0"
 
 if (!semver.satisfies(process.versions.bun, expectedBunVersionRange)) {
   throw new Error(`This script requires bun@${expectedBunVersionRange}, but you are using bun@${process.versions.bun}`)
